src/charts/pie.coffee

```diff
@@ -57,8 +57,7 @@ Ember.Charts.PieComponent = Ember.Charts.ChartComponent.extend(
       value: d.value
       percent: d3.round( 100 * d.value / total )
 
-    key = @get 'sortKey'
-    data.sortBy key
+    _.sortBy data, @get('sortKey')
```

Comment:
Shouldn't data be an `Ember.Array`? http://emberjs.com/api/classes/Ember.Array.html#method_sortBy